data/update.py

```diff
@@ -61,7 +61,7 @@
 GATHER_ANALYTICS_URL = META["data"]["analytics_subdomains_url"]
 GATHER_PARENTS = DOMAINS  # Limit subdomains to set of base domains.
 GATHERERS = [
-  ["censys"],
+  ["censys", "--export"],
```

Comment:
This is using the new Censys Export API feature added in https://github.com/18F/domain-scan/pull/100.